docs/notebook.jl

```diff
@@ -0,0 +1,160 @@
+### A Pluto.jl notebook ###
+# v0.12.21
+
+using Markdown
+using InteractiveUtils
+
+# ╔═╡ 40bf1d04-76d0-11eb-0d98-cf2945aa3fd4
+using DataFrames, MixedModels, MixedModelsSim, Random
+
+# ╔═╡ 09276d18-76d2-11eb-3b91-b77f2d532e82
+md"""
+# Simulation of linear mixed models
+
+This notebook shows the process of simulating responses from a mixed-effects model with crossed random effects for subject (`subj`) and item (`item`).
+The general approach is to create a data frame with the appropriate experimental and grouping factors and with an _inert_ dependent variable (`dv`), meaning that, at this stage `dv` is just random noise.
+
+This model is then used in a parametric bootstrap simulation with non-inert coefficient values (β) and random-effects variances and covariances.
+
+Begin by loading the packages to be used.
+"""
+
+# ╔═╡ 28a121e2-76d3-11eb-0896-43da695be0a4
+md"""
+And initialize a random number generator.
+"""
+
+# ╔═╡ b64a889c-76d0-11eb-1936-5545db219def
+rng = MersenneTwister(8675309);
+
+# ╔═╡ ce3be2ec-76e0-11eb-3fb5-67bd31816bca
+md"""
+Next we set the number of subjects, `sub_n`, and the number of items, `item_n`
+"""
+
+# ╔═╡ 21958b24-76ed-11eb-11a2-41223c4864f8
+begin
+	sub_n = 200
+	item_n = 50
+end;
+
+# ╔═╡ b8fe4dd2-76e2-11eb-3a7b-21e2b17043f7
+md"A between-subject factor, `cond`, with two levels, `easy` and `hard` is specified as"
+
+# ╔═╡ feb89de4-76d0-11eb-115e-89e7ed2fc483
+subj_bt = Dict(:cond => ["easy", "hard"]);
+
+# ╔═╡ eb911720-76e2-11eb-002d-21a35fd7d4b2
+md"These settings are used to create a Table with an _inert_ dependent variable, `dv`, as"
+
+# ╔═╡ 286561a4-76d1-11eb-2211-61d84bc1b9a9
+frm = simdat_crossed(rng, sub_n, item_n, subj_btwn = subj_bt);
+
+# ╔═╡ aa915fd2-76ec-11eb-2cec-bff137f10b18
+md"This data table is in the form of a _row table_, which is a vector of `NamedTuple`s"
+
+# ╔═╡ cb5f962a-76ec-11eb-3b1e-6f7f1f68bca5
+typeof(frm)
+
+# ╔═╡ 6914997e-76e3-11eb-2f2b-0501a82bee66
+md"It is generally best to convert this table to a `DataFrame` for display or summary"
+
+# ╔═╡ 9f327062-76e3-11eb-16dc-7be3a2b1c09f
+df = DataFrame(frm)
+
+# ╔═╡ b0202450-76e3-11eb-3218-8bddeed1c50d
+describe(df)
+
+# ╔═╡ 0fce98d0-76e4-11eb-0ed4-430ae7972fce
+md"Note that if you go back to the entry field of `sub_n` or `item_n` and change the value, the other results are automatically updated"
```

Comment:
Should we add a note this is true *for the notebook* and not in general? I'm a tad worried about a Julia novice coming over for the speed (compared to R) when fitting many models in a power simulation and thinking this is a general statement.

Reply:
At this point I think it might be better to delete that comment entirely.  I think it is better to encourage users to create new simulation objects, like `sim1`, rather than redefining existing objects.